ghost/stripe/lib/services/webhook/CheckoutSessionEventService.js

```
@@ -120,6 +120,7 @@ module.exports = class CheckoutSessionEventService {
     }
 
     async handleSubscriptionEvent(session) {
+        console.log('handleSubscriptionEvent >> session.subscription', session.subscription);
```

ESLint

[error] Unexpected console statement.

(no-console)